pyanaconda/payload/image.py

```diff
@@ -201,6 +202,7 @@ def find_optical_install_media():
 
     :return: a device name or None
     """
+    # FIXME: This is duplicated in SetUpCdromSourceTask.run
```

Comment:
Please, move it to the doc string of the function.

Reply:
Done!